docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
       - vela
     environment:
       VELA_ADDR: 'http://localhost:8080'
-      VELA_WEBUI_ADDR: 'http://ui:80'
+      VELA_WEBUI_ADDR: 'http://localhost:8888'
```

Comment:
I have been wondering how this change has not been pushed to the ui, so i am cool with sneaking it in here 👍